frontend/src/pages/api/stats/total-deposit-amount.js

```diff
@@ -0,0 +1,16 @@
+import StatsApi from "../../../libraries/explorer-wamp/stats";
+
+export default async function (req, res) {
+  try {
+    const totalDepositAmount = await new StatsApi(req).totalDepositAmount();
+    if (totalDepositAmount) {
+      res.send(totalDepositAmount);
+    } else {
+      res.send([]);
```

Comment:
Let's return 425 Too Early response here.
```suggestion
      res.status(425).end();
```